taskiq_redis/list_schedule_source.py

```diff
@@ -185,11 +185,11 @@ async def get_schedules(self) -> List["ScheduledTask"]:
         async with Redis(connection_pool=self._connection_pool) as redis:
             buffer = []
             crons = await redis.lrange(self._get_cron_key(), 0, -1)  # type: ignore
-            logger.debug("Got cron scheduleds: %s", crons)
+            logger.debug("Got cron schedules: %s", crons)
```

Comment:
You can also update it to be 
```suggestion
            logger.debug("Got %d cron schedules", len(crons))
```